build/media-suite_compile.sh

```diff
@@ -2054,10 +2054,71 @@ if [[ $ffmpeg != no ]]; then
     fi
 fi
 
+# static do_vcs just for svn
+do_svn() {
```

Comment:
Well, might as well rename it to "check_mplayer_updates" or something.

Reply:
done